packages/compass-collection/src/components/mock-data-generator-modal/types.ts

```diff
@@ -19,7 +19,7 @@ type MockDataGeneratorInProgressState = {
 
 type MockDataGeneratorCompletedState = {
   status: 'completed';
-  fakerSchema: MockDataSchemaResponse;
+  fakerSchema: Array<FakerSchemaMapping>;
```

Comment:
Very small nit: we prefer compact array syntax in compass codebase

```suggestion
  fakerSchema: FakerSchemaMapping[];
```